db/migrate/20180109222722_create_user_second_factors.rb

```diff
@@ -2,8 +2,8 @@ class CreateUserSecondFactors < ActiveRecord::Migration[5.1]
   def change
     create_table :user_second_factors do |t|
       t.integer :user_id, null: false
-      t.string :method
```

Comment:
@featheredtoast Instead of storing the same string multiple times in each column, I represented the method as an integer in https://github.com/discourse/discourse/pull/5612/commits/14f3594f9f39dce7f6a5dda44287560b0a6e53e8#diff-62eb46e48ba621e1c2c1c91dc9763c1bR4 which also takes up less space.